npm/puppeteer/package.json

```
@@ -39,5 +39,16 @@
     "dist",
     "index.d.ts"
   ],
-  "types": "index.d.ts"
+  "types": "index.d.ts",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/cypress-io/cypress.git"
+  },
+  "homepage": "https://github.com/cypress-io/cypress/blob/develop/npm/puppeteer/#readme",
+  "author": "Cypress App Team",
+  "bugs": "https://github.com/cypress-io/cypress/issues/new?assignees=&labels=npm%3A%20%40cypress%2Fangular&template=1-bug-report.md&title=",
```

The label is for `npm: @cypress/angular`, do we have one for this plugin?

```suggestion
  "bugs": "https://github.com/cypress-io/cypress/issues/new?assignees=&labels=npm%3A%20%40cypress%2Fpuppeteer&template=1-bug-report.md&title=",
```

---

Bad copy-pasta on my part 🤦‍♂️ Fixed in https://github.com/cypress-io/cypress/pull/28408/commits/e64b34ce6ff3b48fcdc583b4c860f99723fd790e and created [a matching label](https://github.com/cypress-io/cypress/labels?q=puppeteer+)